query/graphql/src/main/java/org/codice/ddf/admin/graphql/transform/GraphQLTransformOutput.java

```diff
@@ -200,4 +203,39 @@ public String createOutputObjectFieldTypeName(String fieldTypeName) {
                 transformEnum.getEnumTypeProvider(),
                 outputTypeProvider);
     }
+
+    private String toPrettyString(FunctionField<DataType> funcField,
+            List<ErrorMessage> customMessage) {
+        String functionName = funcField.fieldName();
+        List<Object> args = funcField.getArguments()
+                .stream()
+                .map(Field::getValue)
+                .collect(Collectors.toList());
+
+        return Boon.toPrettyJson(hidePassword(toMap(functionName, args, customMessage)));
+    }
+
+    private Object hidePassword(Object result) {
```

Comment:
nit: We may want to stick with only what is known so far and use `password` here instead of `pass` in order to avoid accidentally filtering out fields we didn't mean to. Thoughts?